olm/olmconfig.yaml

```diff
@@ -18,13 +18,19 @@ description: |-
 
 
   This controller is a component of the [AWS Controller for Kubernetes](https://github.com/aws/aws-controllers-k8s)
-  project. This project is currently in **developer preview**. 
+  project. This project is currently in **developer preview**.
+
+
+  **Pre-Instaliation Steps**
```

Comment:
s/Instaliation/Installation/ :)

Reply:
Nice catch! Looks like I don't get spellcheck in `yaml` thought I did. :(